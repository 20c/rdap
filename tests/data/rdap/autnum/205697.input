{
  "handle": "AS205697",
  "startAutnum": 205697,
  "endAutnum": 205697,
  "name": "ALOJALIA",
  "status": [
    "active"
  ],
  "entities": [
    {
      "handle": "ALOJALIA-MNT",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "ALOJALIA-MNT"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/ALOJALIA-MNT"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "AMS346-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "Angel Martinez Sanchez"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "adr",
            {
              "label": "Avd. Federico Anaya, 52\n37004\nSalamanca\nSPAIN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+34910912131"
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "angel@alojalia.com"
          ]
        ]
      ],
      "roles": [
        "technical"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/AMS346-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "APR41-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "Alejandro Pina Rodriguez"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "adr",
            {
              "label": "Avd. Federico Anaya, 52\n37004 Salamanca\nSPAIN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+34910912131"
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "alejandro@alojalia.com"
          ],
          [
            "org",
            {},
            "text",
            "ORG-IYCS4-RIPE"
          ]
        ]
      ],
      "roles": [
        "administrative"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/APR41-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "ORG-IYCS4-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "ALOJALIA CLOUD S.L."
          ],
          [
            "kind",
            {},
            "text",
            "org"
          ],
          [
            "adr",
            {
              "label": "Avd. Federico Anaya, 52 3C\n37004\nSalamanca\nSPAIN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+34910912131"
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "alejandro@alojalia.com"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/ORG-IYCS4-RIPE"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "RIPE-NCC-END-MNT",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "RIPE-NCC-END-MNT"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "org",
            {},
            "text",
            "ORG-NCC1-RIPE"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "self",
          "href": "https://rdap.db.ripe.net/entity/RIPE-NCC-END-MNT"
        },
        {
          "value": "http://www.ripe.net/data-tools/support/documentation/terms",
          "rel": "copyright",
          "href": "http://www.ripe.net/data-tools/support/documentation/terms"
        }
      ],
      "objectClassName": "entity"
    },
    {
      "handle": "AR37103-RIPE",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "Abuse-C Role"
          ],
          [
            "kind",
            {},
            "text",
            "group"
          ],
          [
            "adr",
            {
              "label": "Avd. Federico Anaya, 52\n37004\nSalamanca\nSPAIN"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "email",
            {
              "type": "email"
            },
            "text",
            "abuse@alojalia.com"
          ],
          [
            "email",
            {
              "type": "abuse"
            },
            "text",
            "abuse@alojalia.com"
          ]
        ]
      ],
      "roles": [
        "abuse"
      ],
      "entities": [
        {
          "handle": "es-alojalia-1-mnt",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "fn",
                {},
                "text",
                "es-alojalia-1-mnt"
              ],
              [
                "kind",
                {},
                "text",
                "individual"
              ],
              [
                "org",
                {},
                "text",
                "ORG-IYCS4-RIPE"
              ]
            ]
          ],
          "roles": [
            "registrant"
          ],
          "links": [
            {
              "value": "https://rdap.db.ripe.net/autnum/205697",
              "rel": "self",
              "href": "https://rdap.db.ripe.net/entity/es-alojalia-1-mnt"
            },
            {
              "value": "http://www.ripe.net/data-tools/support/documentation/terms",
              "rel": "copyright",
              "href": "http://www.ripe.net/data-tools/support/documentation/terms"
            }
          ],
          "objectClassName": "entity"
        }
      ],
      "objectClassName": "entity"
    }
  ],
  "remarks": [
    {
      "description": [
        "================================",
        "Website: alojalia.com",
        "Support: soporte@alojalia.com",
        "Abuse: abuse@alojalia.com",
        "Peering: as205697.peeringdb.com",
        "",
        "GTT",
        "COGENT",
        "COMVIVE",
        "DE-CIX",
        "ESPANIX",
        "ACENS",
        "AKAMAI",
        "AMAZON",
        "CATO",
        "CLOUDFLARE",
        "CORE-BACKBONE",
        "DROPBOX",
        "GOOGLE",
        "HURRICANE",
        "INIT7",
        "MICROSOFT",
        "OVH",
        "PCH",
        "SARENET",
        "TWITCH"
      ]
    }
  ],
  "links": [
    {
      "value": "https://rdap.db.ripe.net/autnum/205697",
      "rel": "self",
      "href": "https://rdap.db.ripe.net/autnum/205697"
    },
    {
      "value": "http://www.ripe.net/data-tools/support/documentation/terms",
      "rel": "copyright",
      "href": "http://www.ripe.net/data-tools/support/documentation/terms"
    }
  ],
  "events": [
    {
      "eventAction": "registration",
      "eventDate": "2017-06-22T09:35:50Z"
    },
    {
      "eventAction": "last changed",
      "eventDate": "2023-06-29T15:39:42Z"
    }
  ],
  "rdapConformance": [
    "nro_rdap_profile_asn_flat_0",
    "cidr0",
    "rdap_level_0",
    "nro_rdap_profile_0"
  ],
  "notices": [
    {
      "title": "Filtered",
      "description": [
        "This output has been filtered."
      ]
    },
    {
      "title": "Whois Inaccuracy Reporting",
      "description": [
        "If you see inaccuracies in the results, please visit:"
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "inaccuracy-report",
          "href": "https://www.ripe.net/contact-form?topic=ripe_dbm&show_form=true",
          "type": "text/html"
        }
      ]
    },
    {
      "title": "Source",
      "description": [
        "Objects returned came from source",
        "RIPE"
      ]
    },
    {
      "title": "Terms and Conditions",
      "description": [
        "This is the RIPE Database query service. The objects are in RDAP format."
      ],
      "links": [
        {
          "value": "https://rdap.db.ripe.net/autnum/205697",
          "rel": "terms-of-service",
          "href": "http://www.ripe.net/db/support/db-terms-conditions.pdf",
          "type": "application/pdf"
        }
      ]
    }
  ],
  "port43": "whois.ripe.net",
  "objectClassName": "autnum"
}