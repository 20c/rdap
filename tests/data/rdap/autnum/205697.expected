{"org_address": "", "org_name": "", "emails": ["abuse@alojalia.com", "angel@alojalia.com"], "name": "ALOJALIA"}
